pkg/storkctl/migration.go

```diff
@@ -180,19 +180,26 @@ func newActivateMigrationsCommand(cmdFactory Factory, ioStreams genericclioption
 				}
 			}
 			for _, ns := range migrationScheduleNamespaces {
-				migrSched, err := storkops.Instance().ListMigrationSchedules(ns)
+				migrationSchedules, err := storkops.Instance().ListMigrationSchedules(ns)
 				if err != nil {
 					util.CheckErr(err)
 					return
 				}
-				for _, migr := range migrSched.Items {
-					migr.Status.ApplicationActivated = true
-					_, err := storkops.Instance().UpdateMigrationSchedule(&migr)
+				for _, migrSched := range migrationSchedules.Items {
+					isMigrSchedRelevant, err := isMigrationScheduleRelevant(migrSched, activationNamespaces)
```

Comment:
Let's rename the function to something like isMigrationScheduleMigratingNamespace

Reply:
Will rename it to `func doesMigrationScheduleMigrateNamespaces(migrSched storkv1.MigrationSchedule, activationNs []string) (bool, error)` 
How does that sound?